shimmingtoolbox/unwrap/unwrap_phase.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" Wrapper to different unwrapping algorithms
+
+"""
+
+
+import numpy as np
+from shimmingtoolbox.unwrap.prelude import prelude
+
+
+def unwrap_phase(complex_array, affine, unwrap_function="prelude", mask=np.array([-1])):
```

Comment:
at this point, i would not have a wrapper, but instead call `unwrap.prelude()` directly. why?
- it is easier to deal about dimensions requirements on a function-by-function basis
- add other checks (type of input: mag/phase vs. complex, etc.)

if we do that, there is not much that the wrapper does, so i would remove it for simplicity